### PR TITLE
Feat: replace byline with bylines

### PR DESCRIPTION
**Pull request by ValGeorgiev**
This PR is replacing **byline** attribute with **bylines**

P.S. Daily Universal Register byline is not migrated in TPA side, so we will use byline for now

### CODE REVIEW

File: packages/article-skeleton/src/article-tracking-context.js [hunk 1]
```diff
@@ -5,7 +5,11 @@ export default Component =>
   withTrackingContext(Component, {
     getAttrs: ({ data, pageSection, referralUrl = "" }) => ({
       articleId: get(data, "id", ""),
-      byline: get(data, "byline[0].children[0].attributes.value", ""),
+      bylines: get(
+        data,
+        "bylines[0].byline[0].children[0].attributes.value",
```
Comment (aronshamash) [inline packages/article-skeleton/src/article-tracking-context.js]: is this what we want? just the first author ?

File: packages/article-magazine-comment/src/article-header/article-header-prop-types.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import PropTypes from "prop-types";
 
 const articleHeaderPropTypes = {
   authorImage: PropTypes.string.isRequired,
-  byline: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+  bylines: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
```
Comment (aronshamash) [inline packages/article-magazine-comment/src/article-header/article-header-prop-types.js]: we use 3 different propTypes shapes in different contexts is this correct?

1. `PropTypes.arrayOf(PropTypes.shape({}))`
2. `PropTypes.arrayOf(PropTypes.shape(nodeShape))`
3. `PropTypes.arrayOf(treePropType)`

File: packages/article-skeleton/src/article-tracking-context.js (hunk 1 above)
Comment (ValGeorgiev) [reply packages/article-skeleton/src/article-tracking-context.js]: yes, I just refactor the old one

File: packages/article-magazine-comment/src/article-header/article-header-prop-types.js (hunk 2 above)
Comment (ValGeorgiev) [reply packages/article-magazine-comment/src/article-header/article-header-prop-types.js]: only the first one is not failing, working on that..., will refactor all contexts, thanks

File: packages/article-magazine-comment/src/article-magazine-comment.js [hunk 3]
```diff
@@ -48,18 +48,13 @@ class ArticleMagazineComment extends Component {
         ? bylines[0].image.crop.url
         : null;
 
-    const authorByline =
-      bylines && bylines.length > 0 && Object.keys(bylines[0]).length !== 0
-        ? bylines[0].byline
-        : null;
-
```
Comment (ValGeorgiev) [inline packages/article-magazine-comment/src/article-magazine-comment.js]: @aronshamash What is your opinion here? Are you fine with this one. In ArticleHeader/Meta component we have such logic

**Comment by tuncaulubilge** [comment]
Let's keep this PR on hold until RDS switch is done. Currently, bylines are not populated on prod

File: packages/article-magazine-comment/src/article-magazine-comment.js (hunk 3 above)
Comment (aronshamash) [reply packages/article-magazine-comment/src/article-magazine-comment.js]: yes fine with it

Comment (nehasri89) [reply packages/article-magazine-comment/src/article-magazine-comment.js]: I don't see this logic in `ArticleHeader/Meta`

File: packages/article-main-comment/__tests__/web/__snapshots__/semantic.web.test.js.snap [hunk 4]
```diff
@@ -20,7 +20,7 @@ exports[`1. a full article 1`] = `
   ADVERTISEMENT
   <img
     alt=""
-    src="https://image.io"
+    src={null}
```
Comment (nehasri89) [inline packages/article-main-comment/__tests__/web/__snapshots__/semantic.web.test.js.snap]: this should still have some value

File: packages/provider-queries/src/article-fragment.js [hunk 5]
```diff
@@ -130,7 +115,21 @@ export default gql`
   }
 
   fragment articleProps on Article {
-    byline
```
Comment (nehasri89) [inline packages/provider-queries/src/article-fragment.js]: does Daily Universal Register byline need it still?

File: packages/provider-queries/src/article.web.js [hunk 6]
```diff
@@ -132,7 +117,21 @@ export default addTypenameToDocument(gql`
   }
 
   fragment articleProps on Article {
-    byline
```
Comment (nehasri89) [inline packages/provider-queries/src/article.web.js]: same as above

File: packages/article-magazine-comment/src/article-magazine-comment.js [hunk 7]
```diff
@@ -48,18 +48,13 @@ class ArticleMagazineComment extends Component {
         ? bylines[0].image.crop.url
         : null;
 
-    const authorByline =
```
Comment (nehasri89) [inline packages/article-magazine-comment/src/article-magazine-comment.js]: I don't see this logic in ArticleHeader/Meta

**Comment by nehasri89** [comment]
LG apart from one small comment

**Pull request merged**